siliconcompiler/apps/sc_dashboard.py

```diff
@@ -54,12 +55,24 @@ def main():
 
     graph_chips = []
     if switches['graph_cfg']:
-        for i, file_path in enumerate(switches['graph_cfg']):
+        for name_and_file_path in switches['graph_cfg']:
+            args = len(name_and_file_path)
+            if args == 0:
+                continue
+            elif args == 1:
+                file_path = name_and_file_path[0]
+                name = file_path
```

Comment:
make the name "cfgN", where N is just an increment of the position the list

Reply:
I don't understand what you mean. Let's discuss.